src/utils/datetime_util.py

```diff
@@ -18,8 +19,8 @@
 START_TIME_FORMAT_NO_TZ = "%I:%M%p"
 DATE_FORMAT = '%Y-%m-%d'
 DATE_TITLE_FORMAT = '%d %b %Y'
-MINUTES_BEFORE_GAME_START_TO_CREATE_POST = 60
-MINUTES_AFTER_GAME_END_TO_UPDATE_POST = 60
+#MINUTES_BEFORE_GAME_START_TO_CREATE_POST = 60  # Now configurable in the environment file
```

Comment:
Delete these instead of commenting them out.

Reply:
> Delete these instead of commenting them out.

Done.